src/huggingface_hub/repository.py

```diff
@@ -265,7 +275,7 @@ def clone_from(self, repo_url: str, use_auth_token: Union[bool, str, None] = Non
 
             # checks if repository is initialized in a empty repository or in one with files
             if len(os.listdir(self.local_dir)) == 0:
-                logger.debug(f"Cloning {clean_repo_url} into local empty directory.")
+                logger.warning(f"Cloning {clean_repo_url} into local empty directory.")
```

Comment:
I'm also taking the opportunity to elevate this log's importance, as without it the user might wonder why the Repo instantiation is taking so long. 

Can ignore this in favor of a slight rework of logs as proposed by @thomwolf